docs/espace/tutorials/digitalSignature/ECDSA.md

```diff
@@ -0,0 +1,160 @@
+---
+displayed_sidebar: eSpaceSidebar
+---
+# How to use ECDSA Signature
+
+ECDSA (Elliptic Curve Digital Signature Algorithm) is a widely used digital signature algorithm in blockchain and cryptocurrency. It has the following characteristics:
+
+- High security: Based on elliptic curve cryptography
+- Short signature length: ECDSA generates more compact signatures compared to algorithms like RSA
+- Fast verification: Suitable for use in smart contracts
+
+### Basic Smart Contract Example
+
+Here's a simple Solidity smart contract demonstrating the basic use of ECDSA signatures:
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+contract SimpleECDSAExample {
+    using ECDSA for bytes32;
+
+    function verifySignature(bytes32 message, bytes memory signature, address signer) public pure returns (bool) {
+        return message.toEthSignedMessageHash().recover(signature) == signer;
```

Comment:
I believe this needs to be changed for:

return message.recover(signature) == signer;

I tested it and this validation doesn't worked for me, but with the change above it does.

Reply:
Sorry, I overlooked that `signMessage()` automatically applies the EIP-191 prefix.